src/Stratis.Bitcoin.Features.Wallet/Models/RequestModels.cs

```diff
@@ -107,6 +107,13 @@ public class WalletHistoryRequest : RequestModel
         public string WalletName { get; set; }
 
         public string AccountName { get; set; }
+
+        public int? Skip { get; set; }
+
+        public int? Take { get; set; }
+
+        [JsonProperty(PropertyName = "q")]
```

Comment:
I would call this `query` rather?

Reply:
q is kind of an API convention for search queries. I think it's ok.

Reply:
This is the value included in the URL?